docs/src/building/management-services.md

```diff
@@ -0,0 +1,49 @@
+# Management services
+
+When running a fleet of systems, it is common to use a central management service. Commonly, these services provide a client to be installed on each system which connects to the central service. Often, the management service requires the client to perform a one time registration. The following example shows how to install the client into a bootc image and run it at startup to register the system.
+
+```Dockerfile
+FROM <bootc base image>
+
+# Typically when using a management service, it will determine when to upgrade the system.
+# So, disable bootc-fetch-apply-updates.timer if it is included in the base image.
+systemctl disable bootc-fetch-apply-updates.timer
+
+# Install the client from dnf, or some other method that applies for your client
+RUN dnf install management-client -y && dnf clean all
+
+# Bake the credentials for the management service into the image
+ARG activation_key=<INSERT KEY HERE>
```

Comment:
I think a general best practice is to leave this empty, that way we can error out later if it's explicitly not set...I did that in the SSH key example.

Hmm...do you think we need this "activation key" bit in this doc?  Just talking about how the service basically takes over the bootc updates?

I'm good with it in there too.

Reply:
removed the <> chunk. I think most of these type of management services will require some type of authentication, and documented how to bake the key into the image is interesting.